apps/graphql/src/dataloaders/bookingsLoader/BookingFlowTypes.js

```diff
@@ -7,12 +7,13 @@ export type ApiRouteStop = {|
     +city_id: string,
   |},
   +when: {|
-    +utc: Date,
-    +local: Date,
+    +utc: number,
```

Comment:
Why `number`? I think we should keep it consistent with structure from search result.

Reply:
Because that is what is what data is actually return
https://github.com/kiwicom/margarita/blob/master/apps/graphql/src/datasets/AllBookings.json#L61

Reply:
I see, I'm sorry I didn't notice the name of the `const`.